.github/workflows/docker_build_push.yml

```
@@ -4,6 +4,8 @@ on:
     branches:
       - master
       - develop
+    paths:
+      - BALSAMIC/containers/
```

Just realised that the syntax may be incorrect, although I'm not sure if it will work with just BALSAMIC/containers/

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

```suggestion
      - "BALSAMIC/containers/**"
```